website/content/docs/enterprise/network-segments.mdx

```diff
@@ -11,19 +11,33 @@ description: |-
 <EnterpriseAlert>
   This feature requires{' '}
   <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>{' '}
-  with the Global Visibility, Routing, and Scale module.
+  version 0.9.3+ with the Global Visibility, Routing, and Scale module.
 </EnterpriseAlert>
 
-Consul Network Segments enables operators to create separate LAN gossip segments
-in one Consul cluster. Agents in a segment are only able to join and communicate
-with other agents in its network segment. This functionality is useful for
-clusters that have multiple tenants that should not be able to communicate
-with each other.
+Consul requires full connectivity between all agents (servers and clients) in a
+[datacenter](/docs/agent/options#_datacenter) within a given
+LAN gossip pool. By default, all Consul agents will be a part of one shared Serf LAN
+gossip pool known as the `<default>` network segment, thus requiring full mesh
+connectivity within the datacenter.
 
-To get started with network segments you can review the tutorial on HashiCorp Learn for
-[Network Segments](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters).
+![Consul datacenter default agent connectivity: one network segment](/img/network-segments/consul-network-segments-single.png)
 
-~> **Note:** Prior to Consul 1.7.3, a Consul agent configured with too many network segments may not be able to start due to [limitations](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters#network-segments-limitations) in Serf.
+In some environments, full connectivity between all agents is not possible—known as a
+"segmented network". This is usually the result of business policies enforced through
+network rules or firewalls. To use Consul in a segmented network, you must break up
+the LAN gossip pool along network communication boundaries into separate "network
+segments". Network segments are isolated LAN gossip pools which require full connectivity
+only between all agent members on the same segment.
+
+![Consul datacenter agent connectivity with network segments](/img/network-segments/consul-network-segments-multiple.png)
+
+To get started with network segments you can review the tutorial on HashiCorp
+Learn for [Network Segments](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters).
+
+-> **Info:** Network segments enable you to operate a datacenter without full mesh
+(LAN) connectivity between all agents. To enable operating multiple federated Consul
+datacenters without full mesh (WAN) connectivity between all server agents, use
+[Network Areas (Enterprise)](/docs/enterprise/federation) instead.
```

Comment:
```suggestion
-> **Info:** Network segments enable you to operate a Consul datacenter without full mesh
(LAN) connectivity between agents. To federate multiple Consul
datacenters without full mesh (WAN) connectivity between datacenters, use
[Network Areas (Enterprise)](/docs/enterprise/federation).
```
Within a given area, servers between peered DCs are fully meshed.

Reply:
Instead of `without full mesh (WAN) connectivity between datacenters`, should it be `... between Consul servers`? The first phrasing implies (to me) that we require full mesh connectivity between all agents across federated datacenters (which I didn't think was true), not just across all server agents.

Reply:
> The first phrasing implies (to me) that we require full mesh connectivity between all agents across federated datacenters.

WAN federation only takes place between servers, not clients.

By default, federated datacenters form a full mesh of connectivity to servers in other DCs on the WAN gossip pool. Network Areas are useful when it is not possible, or not desirable, to have a full mesh of connectivity between all servers in all datacenters.

Within an area, servers establish a full mesh of connectivity to servers in the peer DC. As such, it's correct to say that this feature is useful when there is not full mesh connectivity between all _DCs_, as all servers within the area still require the ability to directly communicate to their peers.

Reply:
> As such, it's correct to say that this feature is useful when there is not full mesh connectivity between all DCs, as all servers within the area still require the ability to directly communicate to their peers.

We're on the same page. Network segments are to agents within a DC as network areas are to server agents across DCs.

I'm just trying to figure out how to embed that concisely in the sentence. If we assume someone already understands clearly how Consul WAN federation works, then I agree "between all datacenters" will be understood. Is that an assumption we want to make here? And if not, is there something else we can say?

e.g., `without full mesh (WAN) connectivity between all server agents in all datacenters` instead of just `between all datacenters`?

Reply:
Changed to:
```
To federate multiple Consul datacenters
without full mesh (WAN) connectivity between all server agents in all datacenters,
```